.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "7.5.0"
+env:
+  - NPM_CONFIG_LOGLEVEL=warn
+install:
+  - npm install
+  - npm run bootstrap
```

Comment:
You're right postinstall does work correctly and `npm run bootstrap` ends up running twice. I'll remove one entry